examples/templates/gcp-devcontainer/main.tf

```
@@ -51,25 +85,48 @@ data "coder_parameter" "zone" {
   }
 }
 
-provider "google" {
-  zone    = data.coder_parameter.zone.value
-  project = var.project_id
-}
-
-data "google_compute_default_service_account" "default" {
+data "coder_parameter" "instance_type" {
+  name         = "instance_type"
+  display_name = "Instance Type"
+  description  = "Select an instance type for your workspace."
+  type         = "string"
+  mutable      = false
+  order        = 2
+  default      = "e2-micro"
+  option {
+    name  = "e2-micro (2C, 1G)"
+    value = "e2-micro"
+  }
+  option {
+    name  = "e2-small (2C, 2G)"
+    value = "e2-small"
+  }
+  option {
+    name  = "e2-medium (2C, 2G)"
+    value = "e2-medium"
+  }
 }
 
-data "coder_workspace" "me" {
+data "coder_parameter" "fallback_image" {
+  default      = "codercom/enterprise-base:ubuntu"
+  description  = "This image runs if the devcontainer fails to build."
+  display_name = "Fallback Image"
+  mutable      = true
+  name         = "fallback_image"
+  order        = 3
 }
-data "coder_workspace_owner" "me" {}
 
-resource "google_compute_disk" "root" {
-  name  = "coder-${data.coder_workspace.me.id}-root"
-  type  = "pd-ssd"
-  image = "debian-cloud/debian-12"
-  lifecycle {
-    ignore_changes = [name, image]
-  }
+data "coder_parameter" "devcontainer_builder" {
+  description  = <<-EOF
+Image that will build the devcontainer.
+We highly recommend using a specific release as the `:latest` tag will change.
+Find the latest version of Envbuilder here: https://github.com/coder/envbuilder/pkgs/container/envbuilder
```

```suggestion
Find the latest version of Envbuilder here: https://ghcr.io/coder/envbuilder
```
Shorter URL, maybe nicer?

---

Oh nice, TIL you could do this!